app/controllers/application_controller.rb

```diff
@@ -1,3 +1,38 @@
 class ApplicationController < ActionController::Base
+  include ApplicationHelper
+  include MessageHelper
```

Comment:
we talked about this in your eval a bit, helper modules are conventionally used for your views - their responsibility is formatting output for viewing purposes. i like how you refactored your controllers - could we make a PORO though to handle some of this logic?